apps/editors/views.py

```diff
@@ -634,6 +678,9 @@ def reviewlog(request):
     form = forms.ReviewLogForm(data)
 
     approvals = ActivityLog.objects.review_queue()
+    if not acl.check_unlisted_addons_reviewer(request):
+        # Only display logs related to unlisted addons to senior reviewers.
```

Comment:
Nit: This is a bit ambiguous. I think it means "Display logs related to unlisted add-ons only to senior reviewers"